Project_For_Binod/SignedInView/ChangePicture/ChangePictureViewModel.swift

```diff
@@ -59,6 +74,9 @@ struct DefaultChangePictureViewModel: ChangePictureViewModel {
   init(userSession: UserSession, userSessionRepository: UserSessionRepository) {
     self.userSession = userSession
     self.userSessionRepository = userSessionRepository
```

Comment:
should make this an interface which is specific to what the signup screen needs rather than a general user session repository, since the user is:

* not logged in
* there should be no session-related functions visible to this class
* the class should only have signin-related functionality visible to it